src/tools/perf/lib/libperf_int.h

```diff
@@ -216,6 +216,17 @@ static inline void ucx_perf_update(ucx_perf_context_t *perf,
     }
 }
 
+static UCS_F_ALWAYS_INLINE unsigned rte_peer_index(unsigned group_size,
+                                                   unsigned group_index)
+{
+    int size  = group_size;
+    int index = group_index;
+    int peer_index = size - 1 - index;
```

Comment:
Do we need to `assert()` here to ensure no overflow?

Reply:
pls see the next line